cron/hourly/git_maintenance_tasks_lock

```diff
@@ -4,4 +4,5 @@ set -e
 # Run git_maintenance_tasks only if not running already
 DIR="$(readlink -e "$(dirname $0)/..")"
 mkdir -p ~/lmms-repo
-flock -w 0 ~/lmms-repo/git_maintenance_tasks.lock "$DIR/git_maintenance_tasks"
+flock -w 0 ~/lmms-repo/git_maintenance_tasks.lock \
+	"$DIR/git_maintenance_tasks_retry"
```

Comment:
>I would prefer we step back as originally proposed and don't use lock files at all .

This has nothing to do with merging scripts. `ps` parsing is inferior to `flock`:
1. It is more complicated.
2. Usage is limited to one instance per process name.

OTOH, you are not limited to pass a command to `flock`. It helps maintenance if you do so.